<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1684754031744" stop="1684754113050" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>ULP_BFF_Test_Suite : ULP_BFF</name>
    <title>ULP_BFF_Test_Suite : ULP_BFF</title>
    <test-cases>
        <test-case start="1684754031868" stop="1684754036441" status="passed">
            <name>API_01_VerifyUserIsAbleToGenerateDigiUrlForPortal</name>
            <title>Verify that user is able to generate digiurl for Portal</title>
            <description type="text">Verify that user is able to generate digiurlfor portal</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="ULP_BFF_Test_Suite"/>
                <label name="testGroup" value="ULP_BFF"/>
                <label name="testClass" value="com.api.script.DigilockerportalTest"/>
                <label name="testMethod" value="API_01_VerifyUserIsAbleToGenerateDigiUrlForPortal"/>
                <label name="host" value="DESKTOP-UJIBV1G"/>
                <label name="thread" value="13592@DESKTOP-UJIBV1G.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1684754036453" stop="1684754103949" status="passed">
            <name>API_02_VerifyUserIsAbleToGenerateAuthCode</name>
            <title>Verify that user is able to generate auth code</title>
            <description type="text">Verify that user is able to generate auth code</description>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="ULP_BFF_Test_Suite"/>
                <label name="testGroup" value="ULP_BFF"/>
                <label name="testClass" value="com.api.script.DigilockerportalTest"/>
                <label name="testMethod" value="API_02_VerifyUserIsAbleToGenerateAuthCode"/>
                <label name="host" value="DESKTOP-UJIBV1G"/>
                <label name="thread" value="13592@DESKTOP-UJIBV1G.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1684754103957" stop="1684754112199" status="broken">
            <name>API_03_VerifyUserIsAbleToGenerateToken</name>
            <title>Verify user is able to generate token</title>
            <description type="text">Verify user is able to generate token</description>
            <failure>
                <message>NullPointerException: Cannot invoke "com.api.pojoResponse.Digi.getAccess_token()" because the return value of "com.api.pojoResponse.DigiLockerTokenEwalletResponse.getDigi()" is null</message>
                <stack-trace>java.lang.NullPointerException: Cannot invoke "com.api.pojoResponse.Digi.getAccess_token()" because the return value of "com.api.pojoResponse.DigiLockerTokenEwalletResponse.getDigi()" is null&#13;
	at com.api.view.APIview.getAccessToken(APIview.java:178)&#13;
	at com.api.script.DigilockerportalTest.API_03_VerifyUserIsAbleToGenerateToken(DigilockerportalTest.java:80)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)&#13;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)&#13;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)&#13;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&#13;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)&#13;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:794)&#13;
	at org.testng.TestRunner.run(TestRunner.java:596)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1063)&#13;
	at org.testng.TestNG.run(TestNG.java:1031)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="ULP_BFF_Test_Suite"/>
                <label name="testGroup" value="ULP_BFF"/>
                <label name="testClass" value="com.api.script.DigilockerportalTest"/>
                <label name="testMethod" value="API_03_VerifyUserIsAbleToGenerateToken"/>
                <label name="host" value="DESKTOP-UJIBV1G"/>
                <label name="thread" value="13592@DESKTOP-UJIBV1G.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1684754112210" stop="1684754112807" status="broken">
            <name>API_01_VerifyUserIsAbleToSearchStudent</name>
            <title>Verify user is able to search student</title>
            <description type="text">Verify user is able to search student</description>
            <failure>
                <message>NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "com.api.pojoResponse.BffCredentialsSearchStudentsresponse.getResult()" is null</message>
                <stack-trace>java.lang.NullPointerException: Cannot invoke "java.util.List.get(int)" because the return value of "com.api.pojoResponse.BffCredentialsSearchStudentsresponse.getResult()" is null&#13;
	at com.api.view.APIview.getCredentialID(APIview.java:432)&#13;
	at com.api.script.DigilockerBFFCredentialsTest.API_01_VerifyUserIsAbleToSearchStudent(DigilockerBFFCredentialsTest.java:57)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)&#13;
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)&#13;
	at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)&#13;
	at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)&#13;
	at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)&#13;
	at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)&#13;
	at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:794)&#13;
	at org.testng.TestRunner.run(TestRunner.java:596)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:276)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1063)&#13;
	at org.testng.TestNG.run(TestNG.java:1031)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="ULP_BFF_Test_Suite"/>
                <label name="testGroup" value="ULP_BFF"/>
                <label name="testClass" value="com.api.script.DigilockerBFFCredentialsTest"/>
                <label name="testMethod" value="API_01_VerifyUserIsAbleToSearchStudent"/>
                <label name="host" value="DESKTOP-UJIBV1G"/>
                <label name="thread" value="13592@DESKTOP-UJIBV1G.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
